.github/workflows/update-contributors.yml

```diff
@@ -0,0 +1,83 @@
+name: allcontributors-auto-detect
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  Update:
+    name: Generate
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Tributors Update      
+        uses: con/tributors@0.0.15
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:        
+
+          # Single text list (space separated) of parsers, leave unset to auto-detect
+          parsers: unset
+
+          # Update lookup with GitHub metadata
+          update_lookup: github
+
+          # Skip these users (example)
+          skip_users: manbat
+
+          # INFO, DEBUG, ERROR, WARNING, etc.
+          log_level: DEBUG
+
+          # If files already exist and an init is done, force overwrite
+          force: true
+
+          # the minimum number of contributions required to add a user
+          threshold: 1
+
+      - name: Checkout New Branch
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          BRANCH_AGAINST: "master"
```

Comment:
That is an interesting one... we have a workflow that bugfixes, and bug fix releases come from `maint` branch which periodically gets merged into `master` (which has all the "bleeding edge" developments towards next release). Should we just monitor that one instead or in addition to `master`?

Reply:
Sure that would work!